fcrepo4/fcrepo4
# Enable integration test framework and running of jetty-console

This update depends on the merge of: https://github.com/fcrepo4-labs/jetty-console/pull/1

An improvement to this PR would be to refactor the pom.xml dependency updates in the top-level "dependencyManagement" section.

**JIRA Ticket**:
https://jira.lyrasis.org/browse/FCREPO-3153

# What does this Pull Request do?
- Updates dependencies
- Allows jetty-console to run

# How should this be tested?
- Build the project with the "one-click" profile
```
mvn clean install -Pone-click
```
- Run the one-click app
```
java -jar fcrepo-webapp/target/fcrepo-webapp-6.0.0-SNAPSHOT-jetty-console.jar --headless
```
   - Observe a running Fedora at port 8080
   - Note: Since many of the Injected components are not yet available, the application fails on any UI action

# Interested parties
Tag @fcrepo4/committers
